templates/user/AccountSettingsSpecialist.html

```diff
@@ -0,0 +1,110 @@
+                        <div class="card no-services-services">
+                            <div class="card_content">
+                                <p class="flow-text center-align services-title">
+                                    You don`t offer any services. You can easily add them.</p>
+                            </div>
+                        </div>
+                    {% endif %}
+                </div>
+                <div class="col m6">
+                    <div class="card" id="search_service_card">
+                        <div class="card-content">
+                            <h5>Feel free to add services you offer</h5>
+                            <div class="input-field">
+                                <input type="text" id="sign_up_service_selector" class="typeahead">
+                                <div id="error_sign_up_service_selector" class="error"></div>
+                            </div>
+                            <div class="row">
+                                <div id="selected_services">
+                                </div>
+                            </div>
+                            <button id="search_service_submit" class="waves-effect waves-light btn grey darken-1 grey-text text-lighten-4">Add selected services</button>
+                        </div>
+                    </div>
+
+                    <div class="card" id="add_service_card">
+                        <div class="card-content">
+                            <h5>Cannot find your service? Add by yourself.</h5>
+                            <form id="add_service_form" method="POST">
+                                {{ ser_form.csrf_token }}
+                                <div class="input-field">
+                                    {{ ser_form.title(id="service_title") }}
+                                    <label for="service_title">Service name</label>
+                                    <div id="error_service_title" class="error"></div>
+                                </div>
+                                <div class="input-field">
+                                    {{ ser_form.domain(id="service_domain") }}
+                                    <label for="service_domain">Domain of service</label>
+                                    <div id="error_service_domain" class="error"></div>
+                                </div>
+                                <div class="input-field">
+                                    {{ ser_form.description(id="service_description", class="materialize-textarea") }}
+                                    <label for="service_description">Description of service (optional)</label>
+                                    <div id="error_service_description" class="error"></div>
+                                </div>
+                                <a id="service_submit" class="waves-effect waves-light btn grey darken-1 grey-text text-lighten-4">Create</a>
```

Comment:
And here.
Maybe it's better to join style-related classes into something, and describe it in css?